osu.Game.Rulesets.tau/Extensions.cs

```diff
@@ -18,7 +18,7 @@ public static float GetDegreesFromPosition(this Vector2 target, Vector2 self)
         public static float GetHitObjectAngle(this Vector2 target)
         {
             Vector2 offset = new Vector2(256, 192) - target; // Using centre of playfield.
-            return (float)MathHelper.RadiansToDegrees(Math.Atan2(-offset.X, -offset.Y));
+            return (float)MathHelper.RadiansToDegrees(Math.Atan2(offset.X, offset.Y));
```

Comment:
```suggestion
            return (float)MathHelper.RadiansToDegrees(Math.Atan2(offset.X, -offset.Y)) - 90;
```
seems like this code works much better